src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -157,14 +157,23 @@ function MultiCombobox<Item>({
     },
     [variant]
   );
-  const removeItem = (item: any) => {
+
+  const removeItem = (item: Item) => {
     onChange(value.filter(i => i !== item));
   };
 
-  const addSelectedItem = (item: any) => {
-    if (item !== null) {
-      onChange([...value, item]);
+  const handleChange = (item: Item | Item[] | null) => {
```

Comment:
With the recent changes, a selected value from the dropdown can be both a single item or a group of items`Item[]`. See the next comment for details.